includes/class-wc-cart.php

```diff
@@ -1938,4 +1938,16 @@ private function reset_totals() {
 		$this->fees_api->remove_all_fees();
 		do_action( 'woocommerce_cart_reset', $this, false );
 	}
+
+	/**
+	 * Returns the hash based on cart contents.
+	 *
+	 * @return string hash for cart content
+	 */
+	public function get_cart_hash() {
+		$cart = $this->session->get_cart_for_session();
+		$hash = $cart ? md5( json_encode( $cart ) ) : '';
+
+		return apply_filters( 'woocommerce_add_to_cart_hash', $hash, $cart );
```

Comment:
If you want to rename a filter such as this, see the `class-wc-deprecated-filter-hooks.php` file and add a new case in there for mapping the old to the new.